.github/workflows/chart_release.yaml

```diff
@@ -27,3 +27,58 @@ jobs:
           config: .github/actions/chart_releaser/cr.yaml
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+  
+  gitops_update:
+    runs-on: ubuntu-latest
+    name: Update GitOps Repo
+    needs:
+      - chart-release
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+      - name: Add SSH key
+        env:
+          DITP_GITOPS_REPO_SECRET: ${{ secrets.DITP_GITOPS_REPO_SECRET }}
+        run: |
+          mkdir -p ~/.ssh
+          echo "$DITP_GITOPS_REPO_SECRET" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan github.com >> ~/.ssh/known_hosts
+      - name: Clone gitops-repo
+        run: |
+          git clone git@github.com:bcgov-c/ministry-gitops-ditp.git
+          cd ministry-gitops-ditp
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+      - name: Lookup latest chart
+        id: chart_version
+        run: |
+          helm repo add vc-authn-oidc https://bcgov.github.io/vc-authn-oidc
+          helm repo update
+          echo "::set output name=APP_VERSION::$(helm search repo vc-authn-oidc --version ^0.2 -ojson | jq '.[0].app_version')"
+          echo "::set output name=CHART_VERSION::$(helm search repo vc-authn-oidc --version ^0.2 -ojson | jq '.[0].version')"
```

Comment:
I sthere a way to specify the version so that we do not need to keep this up-to-date manually? If I am reading this correctly it is looking for versions starting with `0.2` and we could move to `0.3` any time, and this would fail to execute?

Reply:
It's a way to show only versions 0.2 or newer. filtering out anything older.
We do not need to keep updating it.
An older chart will not work, as it does not have the required updates to work with Argo pipeline.

Reply:
Unless we can specify a "greater than" clause I think it might be best to remove, since otherwise we need to maintain the filter in the action if using semantic versioning (for anything other than a patch release)